+ndr/+test/+reader/+ced_smr/automatedTest.m

```diff
@@ -0,0 +1,96 @@
+% Channel found (5/14): e23 of type event
+% Channel found (6/14): e24 of type event
+% Channel found (7/14): e25 of type event
+% Channel found (8/14): e26 of type event
+% Channel found (9/14): e27 of type event
+% Channel found (10/14): e28 of type event
+% Channel found (11/14): e29 of type event
+% Channel found (12/14): text30 of type text
+% Channel found (13/14): mk31 of type mark
+% Channel found (14/14): mk32 of type mark
+function test_readevents_epochsamples_native_marker(test_case)
+  filename = utils_get_example('example.smr');
+  reader = ndr.reader('smr');
+ 
+  % 1. Read 'marker' - many channels
+  [timestamps, data] = reader.readevents_epochsamples_native('marker', [30, 31, 32], { filename }, 1, 0, 100);
+
+  verifyEqual(test_case, size(timestamps), [1, 3]);
+  verifyEqual(test_case, class(timestamps), 'cell');
+  verifyEqual(test_case, size(data), [1, 3]);
+  verifyEqual(test_case, class(data), 'cell');
+
+  t = cell([1 3]);
+  t{1}=[9.4662600000000001;20.25515;31.027609999999999;41.800109999999997;52.572579999999995;63.353249999999996;74.133929999999992;84.91458999999999;95.678879999999992];
+  t{2}=[12.366239999999999;23.146909999999998;33.919399999999996;44.700060000000001;55.472559999999994;66.253209999999996;77.025689999999997;87.806370000000001;98.578849999999989];
+  t{3}=[9.4653299999999998;20.239719999999998;31.016109999999998;41.792499999999997;52.568889999999996;63.345279999999995;74.122669999999999;84.899059999999992;95.675449999999998];
+
+  d = cell([1 3]);
+  d{1}=reshape(char([49 52 56 49 55 57 54 49 50 50 0 0 49 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]), [9 80]);
```

Comment:
This looks scary, but this is actually just:
```
[
    '12                                                                              '
    '4                                                                               '
    '8                                                                               '
    '11                                                                              '
    '7                                                                               '
    '9                                                                               '
    '6                                                                               '
    '1                                                                               '
    '2                                                                               '
]
```    

Would be nice to present it some other way. I've tried a couple, but `verifyEqual` wouldn't take it.
The result above works, and it was created with the uneval tool (https://www.mathworks.com/matlabcentral/fileexchange/19953-uneval-var_name-value-return_cell_array).